pkg/provision/common/zdb.go

```diff
@@ -0,0 +1,38 @@
+package common
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/threefoldtech/zbus"
+	"github.com/threefoldtech/zos/pkg"
+	"github.com/threefoldtech/zos/pkg/stubs"
+)
+
+// DeleteZdbContainer removes a 0-DB container and cleanup all related resources (container, flist, network)
+func DeleteZdbContainer(containerID pkg.ContainerID, zbus zbus.Client) error {
```

Comment:
Any particular reason this function was extracted? it feels to me like it was in the proper place to begin with

Reply:
>// Package common hold logic that is used by both the provision and primitive package
// it purpose is mainly to avoid circular dependencies
package common